docs/tutorials/movement.rst

```diff
@@ -1,19 +1,15 @@
-Movement
+Ruch
 --------
 
-Your BBC micro:bit comes with an accelerometer. It measures movement along
-three axes:
+BBC micro:bit ma wbudowany akcelerometr. Mierzy on ruch wzdłuż trzech osi:
 
-* X - tilting from left to right.
-* Y - tilting forwards and backwards.
-* Z - moving up and down.
+*X - przechylanie od lewej do prawej.
+*Y - odchylanie do przodu i do tyłu.
```

Comment:
Akcelerometr nie mierzy przechyleń (do tego służy żyroskop), tylko przesunięcia.

Reply:
Czytając dalej, widzę, że to autor oryginału nie widział różnicy. Szczerze mówiąc nie jestem pewien co w tej sytuacji zrobić. Chyba zostawić tak, jak jest w oryginale.